server/src/app-dto/enter-attempt.dto.ts

```diff
@@ -18,6 +19,9 @@ export class EnterAttemptDto {
   registrantId?: number;
 
   @IsOptional()
+  @Matches((C.wcaIdRegexLoose), {
+    message: '$value is not a valid WCA ID.',
+  })
```

Comment:
I'm surprised your editor formatted it like this and not as a single line. Do you use Prettier, and does it use the .prettierrc file from this repo?

Reply:
This is also not important

Reply:
I formatted this file once again using the `.prettierrc` from the root folder of the repo, and it's still 3 lines.